nixos/modules/services/networking/teleport.nix

```diff
@@ -74,14 +82,14 @@ in
   };
 
   config = mkIf config.services.teleport.enable {
-    environment.systemPackages = [ pkgs.teleport ];
+    environment.systemPackages = [ cfg.package ];
```

Comment:
@ymatsiuk, you originally introduced the module: is this line required at all? If the service needs `teleport` in PATH (e.g. to re-exec itself), we could add it to the systemd unit path. Otherwise, I think it is conventional to not implicitly mess with `environment.systemPackages` if not necessary.

Reply:
>Otherwise, I think it is conventional to not implicitly mess with `environment.systemPackages` if not necessary.

I can't agree here, even though I have no strong opinion on this, it feels natural to have tools provided by the package when you enable its service, especially if the package provides more than one binary necessary to interact with the running service.